test/api/controllers/routerController/routeMQListener.test.js

```diff
@@ -6,6 +6,7 @@
 var
   should = require('should'),
   winston = require('winston'),
+  q = require('q'),
   params = require('rc')('kuzzle'),
   Kuzzle = require('root-require')('lib/api/Kuzzle'),
```

Comment:
The root-require seems to be missing in the package.json.

However, this module is marked as test only and can be replaced quite easilly using the built-in `require.main.require('./lib/api/Kuzzle')`